test/distributed/_sharded_tensor/ops/test_allclose.py

```diff
@@ -0,0 +1,51 @@
+# Owner(s): ["oncall: distributed"]
```

Comment:
Note that each test file needs to be added to run_test.py. For example: https://github.com/pytorch/pytorch/blob/master/test/run_test.py#L209 (look at all references of test_equals.py) there.